pkg/asset/installconfig/azure/validation_test.go

```diff
@@ -10,10 +10,11 @@ import (
 	aznetwork "github.com/Azure/azure-sdk-for-go/profiles/2020-09-01/network/mgmt/network"
 	azenc "github.com/Azure/azure-sdk-for-go/profiles/latest/compute/mgmt/compute"
 	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/golang/mock/gomock"
```

Comment:
This package is deprecated and https://github.com/uber-go/mock should now be used

#9422 for more context